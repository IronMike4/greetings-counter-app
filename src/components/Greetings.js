import React from "react";

// The Greetings component
const Greetings = () => {
  // Render a paragraph with the text "Hello, World!"
  return <p>Hello, World!</p>;
};

// Export the Greetings component as the default export
export default Greetings;
