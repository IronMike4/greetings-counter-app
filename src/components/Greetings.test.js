import React from "react";
import renderer from "react-test-renderer";
import Greetings from "./Greetings";

// Test to check if the Greetings component renders correctly
test("Greetings component renders correctly", () => {
  // Render the Greetings component to a JSON object
  const tree = renderer.create(<Greetings />).toJSON();
  // Assert that the rendered output matches the previous snapshot
  expect(tree).toMatchSnapshot();
});
