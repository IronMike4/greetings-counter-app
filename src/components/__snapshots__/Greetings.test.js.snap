// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Greetings component renders correctly 1`] = `
<p>
  Hello, World!
</p>
`;
